src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -439,7 +439,7 @@ def transform_summary(self,
         es_response = es_search.execute(ignore_cache=True)
         self._translate_response_aggs(es_response)
         final_response = SummaryResponse(es_response.to_dict())
-        if logger.isEnabledFor(logging.DEBUG):
+        if logger.isEnabledFor(logging.DEBUG) and config.debug == 2:
```

Comment:
```suggestion
        if config.debug == 2 and logger.isEnabledFor(logging.DEBUG):
```

is more natural because the simplest, fastest check should short-circuit other, more complicated checks. Ordering checks this way should be second nature to you, even if the performance difference is insignificant.

I would also like an answer to my previous question. Did you test this in your deployment?

Reply:
Yes, refer to log_id: 35096512501426893458947336217053887030665928500080279570 
in Kibana.
It illustrates the elasticsearch query when `AZUL_DEBUG=2`
There is no logs of the elasticsearch query when `AZUL_DEBUG=1` as intended.

Reply:
Looks good.